sources/packages/api/src/route-controllers/assessment/assessment.aest.controller.ts

```diff
@@ -65,22 +91,28 @@ export class AssessmentAESTController extends BaseController {
       );
 
     if (applicationExceptions.length > 0) {
-      return applicationExceptions.map((applicationException) => ({
-        id: applicationException.id,
-        submittedDate: applicationException.createdAt,
-        status: applicationException.exceptionStatus,
-        requestType: RequestAssessmentTypeAPIOutDTO.StudentException,
-      }));
+      return [].concat(
+        requestAssessmentSummary,
+        applicationExceptions.map((applicationException) => ({
+          id: applicationException.id,
+          submittedDate: applicationException.createdAt,
+          status: applicationException.exceptionStatus,
+          requestType: RequestAssessmentTypeAPIOutDTO.StudentException,
+        })),
+      );
```

Comment:
You can also use the spread operator here results = [...requestAssessmentSummary, applicationExceptions];

Reply:
As requestAssessmentSummary is a value that is a single value from the findOne query, its not an array and the spread operator cannot be used. The spread works for 2 arrays.

Reply:
Ah okay, got you.